src/vdaf.rs

```diff
@@ -447,10 +453,10 @@ where
 
         if outbound.len() == vdaf.num_aggregators() {
             // Another round is required before output shares are computed.
-            inbound = Some(vdaf.prepare_preprocess(outbound.iter().map(|encoded| {
+            inbound = vdaf.prepare_preprocess(outbound.iter().map(|encoded| {
                 V::PrepareMessage::get_decoded_with_param(&states[0], encoded)
                     .expect("failed to decode papare message")
```

Comment:
```suggestion
                    .expect("failed to decode prepare message")
```
But as above, would prefer to propagate the error instead of crashing.